packages/odyssey-react-mui/src/Dialog.tsx

```diff
@@ -111,7 +116,7 @@ const Dialog = ({
       <DialogTitle translate={translate}>
         {title}
         <Button
-          ariaLabel={ariaLabel}
+          ariaLabel={t("close.text")}
```

Comment:
Do we need to fall back to the supplied one for now?
Also, if someone uses this prop, can we give them a warning / error at build time?

Reply:
This translation already exists for `Toast` so, I just used the same one. Not sure about warning at build time. No strong feelings either way

Reply:
I just wonder if we should warn users about this, if we detect they set the prop. We're pretty much dropping the value on the floor now.

Reply:
@chrispulsinelli-okta I'm fine with warning users, if that is what we want to do. How do I best accomplish that?